tutornotes/plugin.py

```diff
@@ -26,18 +29,27 @@
 }
 
 # Initialization hooks
-tutor_hooks.Filters.COMMANDS_INIT.add_item((
-    "mysql",
-    ("notes", "tasks", "mysql", "init"),
-))
-tutor_hooks.Filters.COMMANDS_INIT.add_item((
-    "lms",
-    ("notes", "tasks", "lms", "init"),
-))
-tutor_hooks.Filters.COMMANDS_INIT.add_item((
-    "notes",
-    ("notes", "tasks", "notes", "init"),
-))
+
+# To add a custom initialization task, create a bash script template under:
+# tutorcodejail/templates/codejail/tasks/
+# and then add it to the MY_INIT_TASKS list. Each task is in the format:
+# ("<service>", ("<path>", "<to>", "<script>", "<template>"))
+MY_INIT_TASKS: list[tuple[str, tuple[str, ...], int]] = [
+    ("mysql", ("notes", "tasks", "mysql", "init"), tutor_hooks.priorities.HIGH),
+    ("lms", ("notes", "tasks", "lms", "init"), tutor_hooks.priorities.HIGH),
+    ("notes", ("notes", "tasks", "notes", "init"), tutor_hooks.priorities.HIGH),
```

Comment:
Why are these tasks high priority? Unless there is a good reason to do so, we should add them as normal priority.